modules/cnf-installing-the-operators.adoc

```diff
@@ -0,0 +1,269 @@
+  name: ptp-operators
+  namespace: openshift-ptp
+spec:
+  targetNamespaces:
+    - openshift-ptp
+----
+
+. Create the `03_subscription.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ptp-operator-subscription
+  namespace: openshift-ptp
+spec:
+  channel: "4.6"
+  name: ptp-operator
+  source: "redhat-operators"
+  sourceNamespace: openshift-marketplace
+----
+
+. Configure the `kustomization.yaml` file:
++
+[source,yaml]
+----
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - 01_namespace.yaml
+  - 02_operator_group.yaml
+  - 03_subscription.yaml
+----
+
+== Applying the Stream Control Transmission Protocol (SCTP) patch
+
+Load and enable the blacklisted SCTP kernel module on worker nodes in your cluster.
+
+.Procedure
+
+. Create the `is_ready.sh` executable file:
++
+[source,yaml]
+----
+#!/bin/sh
+
+# no output means that the new machine config wasn't picked by MCO yet
+if [ -z "$(oc get mcp worker-cnf -o jsonpath='{.spec.configuration.source[?(@.name=="load-sctp-module")].name}')" ]; then
+    exit 1
+fi
+
+oc wait mcp/worker-cnf --for condition=updated --timeout 1s
+----
+
+. Create the `sctp_module_mc.yaml` file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker-cnf
+  name: load-sctp-module
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+        - contents:
+            source: data:,
+            verification: {}
+          filesystem: root
+          mode: 420
+          path: /etc/modprobe.d/sctp-blacklist.conf
+        - contents:
+            source: data:text/plain;charset=utf-8,sctp
+          filesystem: root
+          mode: 420
+          path: /etc/modules-load.d/sctp-load.conf
+----
+
+. Configure the `kustomization.yaml` file:
++
+[source,yaml]
+----
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - sctp_module_mc.yaml
+----
+
+== Installing the SR-IOV Network Operator
+
+Install the SR-IOV Network Operator by using the OpenShift Container Platform CLI or the web console.
+
+. Create the ` 01-sriov-namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-sriov-network-operator
+  labels:
+    openshift.io/run-level: "1"
+----
+
+. Create the `02-sriov-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: sriov-network-operators
+  namespace: openshift-sriov-network-operator
+spec:
+  targetNamespaces:
+  - openshift-sriov-network-operator
+----
+
+. Create the `03-sriov-subscription.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sriov-network-operator-subscription
+  namespace: openshift-sriov-network-operator
+spec:
+  channel: "4.6"
```

Comment:
should probably be 4.7 ?

Reply:
Will make this change.

Reply:
Current openshift documentation looks like this:

$ cat << EOF| oc create -f -
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: sriov-network-operator-subsription
  namespace: openshift-sriov-network-operator
spec:
  channel: "${OC_VERSION}"
  name: sriov-network-operator
  source: redhat-operators
  sourceNamespace: openshift-marketplace
EOF

Can we also use "${OC_VERSION} everywhere in the doc?

Reply:
Changed.

Reply:
@StephenJamesSmith It still on 4.6

Reply:
@vitus133 - do we need to change this?

Reply:
I suggest 
```suggestion
  channel: "${OC_VERSION}"
```